.gitignore

```diff
@@ -0,0 +1,8 @@
+.build
+bin/consul_exporter*
```

Comment:
Could you change this to use the `Makefile.COMMON` from https://github.com/prometheus/utils/blob/master/Makefile.COMMON and then this will actually just be called `consul_expoter`. I think the `bin/...` and `prometheus-consul_exporter` lines can go away then.

See https://github.com/prometheus/mesos_exporter/blob/master/Makefile for an example of how to use the common Makefile (which you don't need to modify).

If you prefer, I can also do that in a followup PR, no problem.